docs/themes/hugo-docs/layouts/shortcodes/runnable.html

```diff
@@ -0,0 +1,129 @@
+                    <a class="language" data-action="nav-lang" data-target="curl" href="#">Curl</a>
+                  </nav>
+                </div>
+                <div class="col-2">
+                  <div class="actions pull-right">
+                    <a class="code-btn" data-action="run" href="#">Run</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="editing-header">
+              <div class="row">
+                <div class="col-12">
+                  <span class="pane-title">
+                    Editing query...
+                  </span>
+                </div>
+              </div>
+            </div>
+
+          </div>
+
+          <div class="runnable-content runnable-code">
+            <div class="runnable-tab-content active" data-tab="query">
+              <pre><code class="no-copy"><span class="query-content">{{ $code }}</span></code></pre>
+
+            </div>
+
+            <div class="runnable-tab-content" data-tab="edit">
+              <textarea class="query-content-editable">{{ $code }}</textarea>
+            </div>
+
+            <div class="runnable-tab-content" data-tab="curl">
+                {{ partial "example-curl" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+            </div>
+            <div class="runnable-tab-content" data-tab="go-grpc">
+              {{ partial "example-go-grpc" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+            </div>
+	    <div class="runnable-tab-content" data-tab="java">
+              {{ partial "example-java" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+	    </div>
+            <div class="runnable-tab-content" data-tab="python">
+              {{ partial "example-python" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+            </div>
+            <div class="runnable-tab-content" data-tab="js-grpc">
+              {{ partial "example-js-grpc" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+            </div>
+            <div class="runnable-tab-content" data-tab="js-http">
+              {{ partial "example-js-http" ( dict "Content" . "Code" "blahblah" "Vars" ( .Get "vars" ) ) }}
+            </div>
+          </div>
+
+          <div class="row footer">
+            <div class="col-12">
+              <div class="actions pull-right">
+                <div class="edit-actions">
+                  <a class="code-btn" data-action="reset" href="#">Reset</a>
+                  <a class="code-btn" data-action="discard" href="#">Discard</a>
+                  <a class="code-btn" data-action="save" href="#"><i class="fa fa-save"></i> Save</a>
+                </div>
+                <div class="normal-actions">
+                  <a class="code-btn" data-action="edit" href="#"><i class="fa fa-edit"></i> Edit query</a>
```

Comment:
Why is there a `edit query` button? Actually, why do we even have the runnable file? Badger does not have any runnable code.

Reply:
I have deleted this file along with runnable.css and runnable.js file as we don't require this if we don’t have any runnable code.